test/system/012-manifest.bats

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bats
+
+load helpers
+
+# Regression test for #8931
+@test "podman images - bare manifest list" {
+    # Create an empty manifest list and list images.
+
+    run_podman inspect --insecure --format '{{.ID}}' $IMAGE
+    iid=$output
+
+    run_podman manifest create --insecure test:1.0
+    run_podman images --format '{{.ID}}' --no-trunc
+    [[ "$output" == *"sha256:$iid"* ]]
```

Comment:
Aaaaaaaahhhhhh! Evil! I must never have reviewed the initial commit. Please fix this to
```bash
    assert "$output" =~ "sha256:$iid" "Original image ID still shown in podman-images output"
```